Dependency fix (#23)
achilleas-k/odml-to-nix: README.md

## Dependencies

* Python 3.6+
* Python packages:
    * odml (>=1.4.5)
    * nixio (>=1.5.0b3)

These dependency packages can be manually installed via the python package manager `pip`:

`pip install "odml>=1.4.5" "nixio>=1.5.0b3"` 

or by manually installing the nix-odML-converter from the repository root:

`python setup.py install`

> Python 2 has reached end of life. We will not keep any future versions of nixodmlconverter Python 2 compatible and also recommend using a Python version >= 3.6.

Reviewer: ```suggestion
Python 2 has reached end of life. Future versions of nixodmlconverter will no longer support Python 2.  We further recommend using a Python version >= 3.6.
```
